### ISSUE TITLE
vendor/rubix/ml/src/Exceptions/MIssingExtension.php file has captal I?

### ISSUE THREAD

**Issue by MarkLuds** [andrewdalpino/Rubix-engine#209]
in vendor/rubix/ml/src/Exceptions version 0.4, the file is named MIssingExtension.php, whichthrows the following error:
```
 Uncaught Error: Class 'Rubix\ML\Exceptions\MissingExtension' not found
```
looks like it's the capital i in missing?

**Comment by andrewdalpino** [comment]
Nice catch! I'll go ahead and rename it. Thank you!

**Comment by andrewdalpino** [comment]
Ok this should be fixed now in version `0.4.2`, let me know if this worked. Thank you again!

https://github.com/RubixML/ML/releases/tag/0.4.2

**Comment by MarkLuds** [comment]
Hey, great thanks. Sorry i already upgraded to a newser version. though i did test renaming the file myself before and it worked fine.
